test/db/cmd/cmd_k

```diff
@@ -23,3 +23,14 @@ EXPECT=<<EOF
6","type.int32_t=d","type.int32_t.size=32","type.int64_t=q","type.int64_t.size=64","type.int8_t=b","type.int8_t.size=8","type.long=x","type.long long=q","type.long long.size=64","type.long.size=64","type.pid_t=d","type.pid_t.pid=32","type.short=w","type.short.size=16","type.size_t=d","type.size_t.size=32","type.uid_t=d","type.uid_t.size=64","type.uid_t.uid=32","type.uint16_t=w","type.uint16_t.size=16","type.uint32_t=d","type.uint32_t.size=32","type.uint64_t=q","type.uint64_t.size=64","type.uint8_t=b","type.uint8_t.size=8","type.unsigned char=b","type.unsigned char.size=8","type.unsigned int=i","type.unsigned int.size=32","type.unsigned short=w","type.unsigned short.size=16","type.void *=p","type.void *.size=32","uid_t=type","uint16_t=type","uint32_t=type","uint64_t=type","uint8_t=type","umask=func","ungetc=func","ungetwc=func","unlink=func","unsigned char=type","unsigned int=type","unsigned short=type","user_from_uid=func","usleep=func","vfprintf=func","vfscanf=func","vfwprintf=func","vfwscanf=func","void *=type","vprintf=func","vscanf=func","vsnprintf=func","vsprintf=func","vsscanf=func","vswprintf=func","vswscanf=func","vwprintf=func","vwscanf=func","wait=func","waitid=func","waitpid=func","warn=func","warnc=func","warnx=func","wcrtomb=func","wcscat=func","wcschr=func","wcscmp=func","wcscoll=func","wcscpy=func","wcscspn=func","wcsftime=func","wcslen=func","wcsncat=func","wcsncmp=func","wcsncpy=func","wcspbrk=func","wcsrchr=func","wcsrtombs=func","wcsspn=func","wcsstr=func","wcstod=func","wcstof=func","wcstoimax=func","wcstok=func","wcstol=func","wcstold=func","wcstoll=func","wcstombs=func","wcstoul=func","wcstoull=func","wcstoumax=func","wcsxfrm=func","wctob=func","wctomb=func","wctrans=func","wctype=func","wmemchr=func","wmemcmp=func","wmemcpy=func","wmemmove=func","wmemset=func","wprintf=func","write=func","wscanf=func","xmalloc=func","spec","gcc=spec","spec.gcc.c=char","spec.gcc.d=int","spec.gcc.f=float","spec.gcc.g=double","spec.gcc.ld=long int","spec.gcc.lf=double","spec.gcc.li=long int","spec.gcc.lld=long long int","spec.gcc.lli=long long int","spec.gcc.llu=unsigned long long int","spec.gcc.lu=unsigned long int","spec.gcc.p=void *","spec.gcc.s=const char *","spec.gcc.u=unsigned int","cc","borland=cc","cc.borland.arg0=eax","cc.borland.arg1=edx","cc.borland.arg2=ecx","cc.borland.argn=stack_rev","cc.cdecl-fastcall-ms.argn=stack","cc.cdecl-fastcall-ms.ret=eax","cc.cdecl-thiscall-ms.arg0=ecx","cc.cdecl.argn=stack","cc.cdecl.ret=eax","cc.fastcall.arg0=ecx","cc.fastcall.arg1=edx","cc.fastcall.argn=stack","cc.fastcall.ret=eax","cc.optlink.arg0=eax","cc.optlink.arg1=edx","cc.optlink.arg2=ecx","cc.optlink.argn=stack","cc.optlink.ret=eax","cc.pascal.argn=stack_rev","cc.stdcall.argn=stack","cc.stdcall.ret=eax","cc.watcom.arg0=eax","cc.watcom.arg1=edx","cc.watcom.arg2=ebx","cc.watcom.arg3=ecx","cc.watcom.argn=stack","cdecl=cc","cdecl-thiscall-ms=cc","default.cc=cdecl","fastcall=cc","optlink=cc","pascal=cc","stdcall=cc","watcom=cc","zigns","classes"]}}
```

Comment:
that's a pretty large line can you replace `kj` with `kj~{}` to get that indented? so at least it's readable? ideally i would like to have something shorter. also i think the argument passed to `kj` is ignored. i can't get a partial output like it's possible with just `k anal/cc/*`

Reply:
What to do about the problem with the argument?